Workiva/w-mobile-kit
# MOBILE-1658: Implement drawer to VxD spec

[VxD spec](https://drive.google.com/a/workiva.com/file/d/0B_bu90wZe-wyV0kycHVSVThvZXM/view)
## Description
- Implement the drawer to the VxD spec linked above.
## What Was Changed
- Added a drop shadow.
- Added the ability to specify back and drawer icons.
- Added a border color..
## Testing
1. Ensure that this branch is checked out in the wdesk-ios project.
2. Make sure the drawer looks like it does in the VxD spec above.

---

Please Review: @Workiva/mobile
